tools/pmrfc5424.c

```diff
@@ -245,7 +245,11 @@ CODESTARTparse
 	 */
 
 	/* TIMESTAMP */
-	if(datetime.ParseTIMESTAMP3339(&(pMsg->tTIMESTAMP),  &p2parse, &lenMsg) == RS_RET_OK) {
+	if(lenMsg >= 2 && *p2parse == '-' && *(p2parse+1) == ' ') {
```

Comment:
I think it'll be clearer as `p2parse[0] == '-' && p2parse[1] == ' '` (similar to line 225 above).